floris/tools/floris_interface.py

```diff
@@ -809,6 +810,19 @@ def get_farm_AEP_wind_rose_class(
                 The relative turbine yaw angles in degrees. If None is
                 specified, will assume that the turbine yaw angles are all
                 zero degrees for all conditions. Defaults to None.
+            turbine_weights (NDArrayFloat | list[float] | None, optional):
```

Comment:
These type hints are inconsistent with the type in turbine_weights above (`dtype=int`).